.changeset/curvy-planets-wash.md

```diff
@@ -0,0 +1,6 @@
+---
+"@aws-amplify/ui-react-native": patch
```

Comment:
This is a `changeset-preset` folder under `.github`, you probably need to update something there as well

Reply:
I'll check in with @wlee221 on that. Thinking that we can just delete the `"@aws-amplify/ui-react-native": patch` line but keep the changeset since this PR updates the `ui` package

Reply:
Sounds like a plan :D